README.md

```diff
@@ -1,2 +1,29 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+![FlowChart](Flowchart.png)
```

Comment:
Flowchart looks good 👍  Only minor comment is that "Populate Terminal" is a little vague